scipy/sparse/tests/test_dok.py

```diff
@@ -0,0 +1,30 @@
+from numpy.testing import assert_array_equal
+from scipy.sparse import dok_array
+
+
+def test_fromkeys_default():
+    # test with default value
+    edges = [(0, 2), (1, 0), (2, 1)]
+    Xdok = dok_array.fromkeys(edges)
+    assert_array_equal(
+        Xdok.todense(),
+        [[0, 0, 1], [1, 0, 0], [0, 1, 0]],
+    )
+
+def test_fromkeys_positional():
+    # test with positional value
+    edges = [(0, 2), (1, 0), (2, 1)]
+    Xdok = dok_array.fromkeys(edges, -1)
+    assert_array_equal(
+        Xdok.todense(),
+        [[0, 0, -1], [-1, 0, 0], [0, -1, 0]],
+    )
+
+def test_fromkeys_kwargs():
+    # test with kwarg value
+    edges = [(0,2), (1,0), (2, 1)]
+    Xdok = dok_array.fromkeys(edges, value=-1)
+    assert_array_equal(
+        Xdok.todense(),
+        [[0, 0, -1], [-1, 0, 0], [0, -1, 0]],
+    )
```

Comment:
we only need one of these two tests

Reply:
I added this test because normally fromkeys doesn't take keywords args. Can be deleted if you think it's not required

Reply:
Finally deleted as suggested